dts/bindings/gpio/ite,it8xxx2-gpio-v2.yaml

```diff
@@ -27,6 +27,19 @@ properties:
   wuc-mask:
     type: array
 
+  keyboard-controller:
+    type: boolean
+    description: |
+      gpioksi, gpioksoh and gpioksol require extended setting
+      when used as GPIO output.
```

Comment:
Now that this is a boolean property, I think the following text is a better description:

"When set, this GPIO controller has pins associated with the keyboard controller.  In this
case the reg_gpcr property is overloaded and used to write the keyboard GCTRL register."

Reply:
Done.